src/plugins/janus_sip.c

```diff
@@ -875,6 +875,9 @@ static GThread *handler_thread;
 static void *janus_sip_handler(void *data);
 static void janus_sip_hangup_media_internal(janus_plugin_session *handle);
 
+#define JANUS_RTP_HEADER_LENGTH 12
```

Comment:
This is not needed, we have a define for that already in `rtp.h`.

Reply:
I removed this line in my last commit.